ruby-openai.gemspec

```diff
@@ -25,7 +25,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "httparty", ">= 0.18.1"
+  spec.add_dependency "faraday", "2.7.4"
+  spec.add_dependency "faraday-typhoeus", "1.0.0"
```

Comment:
I personally have no interest in using or install typhoeus on my system.  The advantage of Faraday is you can leave those decisions to someone else.

If you want to test Faraday with typhoeus in the gem, the pattern for that would be to put typhoeus in the Gemfile but not the gemspec, that way it's not required.

Also, our Faraday requirement in our app

```
gem "faraday", ">= 2"
```

We happen to be on 2.7.4, but I'd prefer not to be locked in to that version

```diff
-  spec.add_dependency "faraday", "2.7.4"
+  spec.add_dependency "faraday", ">= 2"
```

Reply:
Great points, thanks for review! Experimenting with Faraday only version.